helm/ffc-workstream/templates/resourceQuota.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: ResourceQuota
+metadata:
+  name: {{ required "name required" .Values.name }}
+  namespace: {{ required "name required" .Values.namespace }}
+  labels:
```

Comment:
Use the library template for labels here instead of copying them: `{{- include "ffc-helm-library.labels" . | nindent 4 }}`